media/asciidoctor-editor.css

```diff
@@ -1065,7 +1065,10 @@ body.toc2 #header>h1:nth-last-child(2) {
   margin-bottom: 1.25em;
   padding: 1.25em;
   /* DISABLED background: #f8f8f7; */
-  background: white; /* TODO: fix transparent toc background */
+  /* Use color variable defined by theme to follow with theme changes
+   * Colors are defined here: https://code.visualstudio.com/api/references/theme-color#editor-colors
+   */
+  background: var(--vscode-editor-lineHighlightBackground);
```

Comment:
Why don't we use `editor.background` ?

Reply:
The previous value let the ToC stand out a bit. The choice for this value goes in the same direction (although there might be better ones).

`background: var(--vscode-editor-lineHighlightBackground);`:
<img width="868" alt="grafik" src="https://github.com/asciidoctor/asciidoctor-vscode/assets/10586564/eb93e749-11ce-4985-8fd6-fcd7875bc853">


`background: var(--vscode-editor-background);`:
<img width="873" alt="grafik" src="https://github.com/asciidoctor/asciidoctor-vscode/assets/10586564/fb8479b7-1aee-4a98-a85f-3577fa6d2d8f">

Reply:
You convinced me!
Let's use your first proposal! Thanks for testing (and showing) each proposal, it's greatly appreciated 👍🏻